packages/server/src/types/feed-items.js

```diff
@@ -0,0 +1,53 @@
+const graphql = require('graphql');
+const { DateType } = require('graphql-sequelize');
+
+const { FEED_ITEM_BODY } = require('../models/enums/feed-item-body');
+
+const feedItemBodyCommon = new graphql.GraphQLObjectType({
+  name: 'FeedItemBodyCommon',
+  fields: () => ({
+    item: { type: graphql.GraphQLNonNull(graphql.GraphQLString) },
+    description: { type: graphql.GraphQLNonNull(graphql.GraphQLString) },
+  }),
+});
+
+const feedItemBody = new graphql.GraphQLObjectType({
+  name: 'FeedItemBody',
+  fields: () => ({
+    [FEED_ITEM_BODY.common]: { type: feedItemBodyCommon },
+  }),
+});
+
+const feedItem = new graphql.GraphQLObjectType({
+  name: 'FeedItem',
+  fields: () => ({
+    id: { type: graphql.GraphQLNonNull(graphql.GraphQLString) },
+    channelId: { type: graphql.GraphQLNonNull(graphql.GraphQLString) },
+    body: { type: graphql.GraphQLNonNull(feedItemBody) },
```

Comment:
I don't know if I like the name ``body``. I'd expect that to be a description or similar.

Maybe we eventually have an enum for ``format``, like "MEASURABLE_CREATED | MEASUREMENT_CREATED..." or similar. Then this field could be called ``formatSpecificDetails`` or just ``formatSpecific``

Reply:
Yes, really, have a sense for me too, but what is better way of naming "body". It is like an aggregation field of similar